langwatch/src/optimization_studio/components/Publish.tsx

```diff
@@ -96,6 +107,44 @@ export function Publish({ isDisabled }: { isDisabled: boolean }) {
   );
 }
 
+const exportWorkflow = async (
+  publishedWorkflow: Workflow,
+  datasetData?: Dataset & { datasetRecords: DatasetRecord[] }
+) => {
```

Comment:
It's not really immediately clear why we're passing in a dataset here. It's so you can attach data to a workflow in the entry node? Maybe add some comments/documentation so the interface and use is clear.